src/Arcus.Messaging.Pumps.ServiceBus/KeyRotation/DefaultAzureServiceBusManagementAuthentication.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Security.Authentication;
+using System.Threading.Tasks;
+using Arcus.Security.Core;
+using GuardNet;
+using Microsoft.Azure.Management.ServiceBus;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.Rest;
+
+namespace Arcus.Messaging.Pumps.ServiceBus.KeyRotation
```

Comment:
This is bigger than just rotation as well:
```suggestion
namespace Arcus.Messaging.Pumps.ServiceBus
```